bastionzero/service/targets/database.go

```diff
@@ -48,12 +49,13 @@ type CreateDatabaseTargetResponse struct {
 
 // ModifyDatabaseTargetRequest is used to modify a Database target
 type ModifyDatabaseTargetRequest struct {
-	TargetName    *string `json:"targetName,omitempty"`
-	ProxyTargetID *string `json:"proxyTargetId,omitempty"`
-	RemoteHost    *string `json:"remoteHost,omitempty"`
-	RemotePort    *Port   `json:"remotePort,omitempty"`
-	LocalPort     *Port   `json:"localPort,omitempty"`
-	LocalHost     *string `json:"localHost,omitempty"`
+	TargetName         *string `json:"targetName,omitempty"`
+	ProxyTargetID      *string `json:"proxyTargetId,omitempty"`
+	ProxyEnvironmentID string  `json:"proxyEnvironmentId,omitempty"`
```

Comment:
```suggestion
	ProxyEnvironmentID *string  `json:"proxyEnvironmentId,omitempty"`
```

All fields in PATCH/`ModifyXYZ` requests should be pointer types with the `omitempty` tag

Reply:
[c909be9](https://github.com/bastionzero/bastionzero-sdk-go/pull/42/commits/c909be9fd7879761e2822e9efbbc2c3e6c8e90cb)